File: beecrowd/Python/iniciante/1012.py
# Área
a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

pi = 3.14159

# formula A = b*h/2
triangulo = (a * c) / 2
# formula (A = π r²)
cirulo = pi * (c**2)
# formula A=(a+b)h/2
trapezio = (a+b) * (c/2)
# formula A = b. h
quadrado = b ** 2
# formula A=b⋅h
retangulo = a * b

print(f"TRIANGULO: {triangulo:.3f}\nCIRCULO: {cirulo:.3f}\nTRAPEZIO: {trapezio:.3f}\nQUADRADO: {quadrado:.3f}\nRETANGULO: {retangulo:.3f}")

File: beecrowd/Cpp/iniciante/1012.cpp
#include <iostream>
#include <iomanip>

int main(){
    double a, b, c;
    double pi = 3.14159;
    double TRIANGULO, CIRCULO, TRAPEZIO, QUADRADO, RETANGULO;

    std::cin >> a;
    std::cin >> b;
    std::cin >> c;

    // formula A = b*h/2
    TRIANGULO = (a * c) / 2;
    // formula (A = π r²)
    CIRCULO = pi * (c * c);
    // formula A=(a+b)h/2
    TRAPEZIO = ((a + b) * c) / 2;
    // formula A = b. h
    QUADRADO = b * b;
    // formula A=b⋅h
    RETANGULO = a * b;

    std::cout << std::fixed << std::setprecision(3);
    
    std::cout << "TRIANGULO: " << TRIANGULO << std::endl;
    std::cout << "CIRCULO: " << CIRCULO << std::endl;
    std::cout << "TRAPEZIO: " << TRAPEZIO << std::endl;
    std::cout << "QUADRADO: " << QUADRADO << std::endl;
    std::cout << "RETANGULO: " << RETANGULO << std::endl;

    return 0;
}